examples/UbidotsSubscribeContext/UbidotsSubscribeContext.ino

```diff
@@ -0,0 +1,142 @@
+/****************************************
+ * Include Libraries
+ ****************************************/
+
+#include <math.h>
+#include <UbidotsMQTT.h>
+
+/****************************************
+ * Define Constants
+ ****************************************/
+
+#ifndef TOKEN
+#define TOKEN "token"  // Ubidots token
+#endif
+
+#define VARIABLE_LABEL "my-var"
+#define DEVICE_LABEL "my-dev"
+
+// Main loop execution period in seconds
+#define LOOP_PERIOD_SEC 5
+
+/****************************************
+ * Globals
+ ****************************************/
+
+UbidotsMQTT clientMQTT(TOKEN, callback);
+
+char ssid[32 + sizeof("")] = "";
+char bssid[2*6 + 5 + sizeof("")] = "";
+char context[64 + sizeof("")] = "";
+
+/****************************************
+ * User Functions
+ ****************************************/
+
+void callback(char* topic, uint8_t* payload, unsigned int length) {
```

Comment:
To make it better to follow the code to the user, just declare the functions and the body of it define it after the loop function.

There will be less code to go through to get where the user would like to go. 

The Documentation should follow the standard https://developer.lsst.io/cpp/api-docs.html

Reply:
Ok!